src/ResolvedApi.ts

```diff
@@ -0,0 +1,236 @@
+import { RequestHandler, RequestCallback } from './request';
+import { ApiCache } from './cache';
+import { Experiment, Experiments } from './experiments';
+import { SearchForm, Form } from './form';
+import Predicates from './Predicates';
+import Cookies from './Cookies';
+import ApiSearchResponse from './ApiSearchResponse';
+import HttpClient from './HttpClient';
+import { Client } from './client';
+
+export const PREVIEW_COOKIE = 'io.prismic.preview';
+export const EXPERIMENT_COOKIE = 'io.prismic.experiment';
+
+export interface Ref {
+  ref: string;
+  label: string;
+  isMasterRef: string;
+  scheduledAt: string;
+  id: string;
+}
+
+export interface ApiData {
```

Comment:
dommage de dupliquer ca. surtout que celui ci est clean alors que l'autre a les quickroutes. A unifier je pense.

Reply:
There is only `ApiData`. I don't see what you mean.

Reply:
Are you comparing `ResolvedApi` and `ApiData` ?

Reply:
okey